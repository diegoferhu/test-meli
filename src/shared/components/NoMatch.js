import * as React from "react";

export default function NoMatch() {
  return <h2>404 Not Found</h2>;
}
