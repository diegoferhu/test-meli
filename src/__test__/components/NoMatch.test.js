import "jsdom-global/register";
import React from "react";
import Enzyme from "enzyme";
import NoMatch from "../../shared/components/NoMatch";

describe("NoMatch", () => {
  test("should render NoMatch component", () => {
    const wrapper = Enzyme.mount(<NoMatch />);
    expect(wrapper.find("h2").text()).toBe("404 Not Found");
    expect(wrapper).toMatchSnapshot();
  });
});
