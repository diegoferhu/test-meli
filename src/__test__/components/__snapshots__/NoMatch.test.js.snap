// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch should render NoMatch component 1`] = `ReactWrapper {}`;
